projects/packages/forms/src/blocks/field-phone/country-list.js

```diff
@@ -0,0 +1,1677 @@
+import { __ } from '@wordpress/i18n';
+
+export const countries = [
```

Comment:
Would it make sense if this list came from the backend?

Reply:
Same as above, need to sort this out :)